### ISSUE TITLE
Q: Planning to write a new carplay support library？

### ISSUE THREAD

**Issue by zhushenwudi** [RedC4ke/flutter_carplay_plus#1]
We are suffering from the fact that the original author of flutter_carplay has given up support and needs a full life cycle scheme. At present, in my project, if the carplay program is started before the app, it will cause abnormal initialization of the flutter engine, and eventually make the layout of the interface font size and so on chaotic. Hopefully, a new creation will solve all the existing pain points.

**Comment by RedC4ke** [comment]
Yes, I was going to rewrite the library from scratch and fix those numerous issues along the way, I don't have much time currently tho. Pull requests are always welcome, I need to rewrite the architecture on Swift side 'cause I went a bit the wrong way since I'm not very familiar with this native environment.

**Comment by RedC4ke** [comment]
You can check out my fork of the original repo, but it was patched up on the go and has many undocumented caveats, for example images work a bit differently (you have to provide a web url)

**Comment by zhushenwudi** [comment]
Well, the Swift architecture is important and requires careful consideration. Its refresh and listening logic may be a breakthrough for new libraries, as well as considerations for the entire lifecycle
At present, I am using my own fork library, which is considered normal use. Only some specific scenarios may have UI display abnormalities
